components/sidebar/sidebar_channel_button_or_link/sidebar_channel_button_or_link.jsx

```diff
@@ -41,6 +41,22 @@ export default class SidebarChannelButtonOrLink extends React.PureComponent {
     constructor(props) {
         super(props);
         this.gmItemRef = React.createRef();
+        this.displayNameRef = React.createRef();
+    }
+
+    state = {
+        showTooltip: false,
+    }
+
+    componentDidMount() {
+        this.onMounted();
+    }
+
+    onMounted = () => {
```

Comment:
I don't get why you create an extra function that is only called from `componentDidMount` that is called `onMounted`, shouldn't be it just in the `componentDidMount`?

Reply:
Thanks for the feedback @jespino!

There is a linting rule that discourages calling `setState` inside `componentDidMount`, it is Ok if it is in a call back function or in another function like `onMounted`. And i thought this approach is better than disabling the rule.